regression9.py

```diff
@@ -0,0 +1,19 @@
+# create Fibonacci Recursion
+list=[]
+def FibRecursion(rnumb):
+    if rnumb == 1:
+        return 0
+    elif rnumb ==2:
+        return 1
+    else:
+        nfib = FibRecursion(rnumb-1)+FibRecursion(rnumb-2)
```

Comment:
You can avoid creating a new variable here by using return statement right here:
`return FibRecursion(rnumb-1)+FibRecursion(rnumb-2)`